.github/workflows/development.yml

```diff
@@ -0,0 +1,147 @@
+# This GHA is responsible for APIM `dev`` deployment.
+# Deployment is initiated using `az cli` bash script.
+#
+# Standard Azure naming convention has been followed:
+# https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming
+#
+#
+# Following Azure services are consumed:
+# 1. Azure resource group - https://learn.microsoft.com/en-us/cli/azure/group?view=azure-cli-latest#az-group-create
+# 2. Azure container registry - https://learn.microsoft.com/en-us/cli/azure/acr?view=azure-cli-latest#az-acr-create
+# 3. Azure container app environment - https://learn.microsoft.com/en-us/azure/container-apps/environment
+# 4. Azure container app - https://learn.microsoft.com/en-us/azure/container-apps/containers
+# 
+#
+# Execution
+# *********
+# GHA is only invoked when following conditions are satisfied:
+# 1. Push to the `dev` branch only.
+# 2. Any modifications inside `src` directory
+
+name: Deployment 🚀
+run-name: APIM dev deployment for ${{ github.repository }}
+
+on:
+  push:
+    branches: [ APIM-149-ia-c-dev-environment ]
```

Comment:
I assume we need to change the branch name here

Reply:
Yes once VPN has been configured.

Reply:
I think this has been updated to `infrastructure` now - can we resolve this?